src/actions/channels.ts

```diff
@@ -823,6 +823,37 @@ export function getChannels(teamId: string, page = 0, perPage: number = General.
     };
 }
 
+export function getArchivedChannels(teamId: string, page = 0, perPage: number = General.CHANNELS_CHUNK_SIZE): ActionFunc {
+    return async (dispatch: DispatchFunc, getState: GetStateFunc) => {
+        dispatch({type: ChannelTypes.GET_CHANNELS_REQUEST, data: null}, getState);
```

Comment:
Unless you're specifically looking at the state in `state.requests.channels.getChannels`, you can remove these lines that dispatch the `GET_CHANNELS_*` actions. We used to use them to track every type of request status, but we found that we don't use them very often